### DISCUSSION TITLE
Can I set a session time out for GUI?

### DISCUSSION THREAD

**Discussion by wwaechter** [jhuckaby/Cronicle#774] [Q&A] [answered]
For security reasons I need to logout the user from GUI after xx minutes inactivity. 
Is it possible? And in wich way i can configure it?

**Comment by jhuckaby** [comment, marked as answer]
The session expiration is expressed as a number of days, so the lowest it can go is 1 day:

https://github.com/jhuckaby/Cronicle/blob/cebf921cb5c90ef66472e1381498b3f2121aab0b/sample_conf/config.json#L117

```js
	"User": {
		"session_expire_days": 1,
```

I'll add a more finite session expiration in Cronicle v2 (coming out later this year).

**Reply by wwaechter** [reply]
I've already set the session_expire_days on 1 day and I think the security department can wait for the next version to get it shorter :o).
